composer.json

```diff
@@ -20,10 +20,7 @@
         "jakub-onderka/php-console-highlighter": "0.3.*"
     },
     "require-dev": {
-        "phpunit/phpunit": "~3.7|~4.0",
-        "symfony/finder": "~2.1",
-        "squizlabs/php_codesniffer": "~2.0",
-        "fabpot/php-cs-fixer": "~1.3"
```

Comment:
StyleCI does have a new feature coming up where by you will be able to run a command line tool that will upload the code to our servers, analyse it, and then download and apply the fixes.